src/com/goide/psi/impl/GoTypeUtil.java

```diff
@@ -41,6 +41,13 @@ private static boolean isArray(@Nullable GoType type) {
 
   public static boolean isString(@Nullable GoType type) {
     GoType underlyingType = type != null ? type.getUnderlyingType() : null;
-    return underlyingType != null && underlyingType.textMatches("string") && GoPsiImplUtil.builtin(underlyingType);
+    return underlyingType != null &&
+           underlyingType.getText() != null &&
```

Comment:
I've changed this to check if the type is not `instanceof GoCType` and added a test for this but it's still in the placeholder test suite, it isn't contained on its own suite. Should I do that instead?